<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugExplorerClass</class>
 <widget class="QMainWindow" name="DebugExplorerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DebugExplorer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWInstances">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Hierarchy</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QPushButton" name="pBtnRefreshHierarchy">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="DebuggerUI.qrc">
               <normaloff>:/DebuggerUI/resources/refresh.png</normaloff>:/DebuggerUI/resources/refresh.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="DebugFilter" name="filterHierarchy" native="true"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QTreeView" name="tWHierarchy">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Classes</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QPushButton" name="pBtnRefreshClass">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="DebuggerUI.qrc">
               <normaloff>:/DebuggerUI/resources/refresh.png</normaloff>:/DebuggerUI/resources/refresh.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="DebugFilter" name="filterClasses" native="true"/>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QTreeView" name="tWClasses">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="pBtnRefreshProp">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="DebuggerUI.qrc">
            <normaloff>:/DebuggerUI/resources/refresh.png</normaloff>:/DebuggerUI/resources/refresh.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="DebugFilter" name="filterProp" native="true"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTableView" name="tbProps">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DebugFilter</class>
   <extends>QWidget</extends>
   <header>debugfilter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DebuggerUI.qrc"/>
 </resources>
 <connections/>
</ui>
